packages/workbox-cli/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I am not 100% sure about the values to use here. If you get caught up, @philipwalton might be able to provide some guidance.